### PR TITLE
fix Makefile bug and update deps for checking bugfix

### PR DESCRIPTION
**Pull request by kpango**
### Description:
SSIA

### Versions:

- Go Version: 1.20.3
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 2.0.9

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/main/CONTRIBUTING.md) document and completed [our CLA agreement](https://cla-assistant.io/vdaas/vald).
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?

### CODE REVIEW

File: internal/info/info.go [hunk 1]
```diff
@@ -252,6 +252,16 @@ func (d Detail) String() string {
 		if maxlen < l {
 			maxlen = l
 		}
+		if tag == "cgo enabled" {
```
Comment (hlts2) [inline internal/info/info.go:255]:
```suggestion
		if tag == "cgo_enabled" {
```

Comment (kpango) [reply internal/info/info.go:255]: it replaced by L250

**Review by hlts2 — APPROVED** [review]
LGTM

**Review by hlts2 — APPROVED** [review]
LGTM

**Review by hlts2 — APPROVED** [review]
LGTM

**Pull request merged**